src/main/routing/TopLevelRoutes.tsx

```diff
@@ -62,10 +60,10 @@ export const TopLevelRoutes: FC = () => {
           }
         />
         <Route
-          path={ROUTE_HOME}
+          path={TopLevelRoutePath.Home}
           element={
             <NonIdentity>
-              <WithApmTransaction path={ROUTE_HOME}>
+              <WithApmTransaction path={TopLevelRoutePath.Home}>
                 <HomePage />
               </WithApmTransaction>
             </NonIdentity>
```

Comment:
I'd suggest the Space route below, to be placed on the bottom of the list after all static/reserved paths.